src/cypress/integration/pages/TradePage.js

```diff
@@ -0,0 +1,79 @@
+class TradePage {
+  getSymbol(symbol) {
+    return cy.get('div[data-testid="tile-' + symbol.replace('/', '') + '"]')
+  }
+  enterNotional(symbol) {
+    return cy.get('div[data-testid="tile-' + symbol + '"] input')
+  }
+
+  getCurrentPrice(symbol, direction) {
+    return cy.get(
+      'div[data-testid="tile-' +
+        symbol +
+        '"] > div > div > div:nth-child(2)  > div:nth-child(2) > div > button[direction=' +
+        direction +
+        '] > div > div'
+    )
+  }
+
+  getDirectionButton(symbol, direction) {
+    return cy.get(
+      'div[data-testid="tile-' +
```

Comment:
Again, these selectors are too brittle.

`div[data-testid="tile-${symbol}"] button[direction="${direction}"]` would be sufficient.

Reply:
`div[data-testid="tile-' + symbol + '"]` is also used a lot throughout. Extracting this to a utli function would be a good idea.

Reply:
Both are fixed in latest commit